### PR TITLE
How to get sources

### PR DESCRIPTION
**Pull request by pdehaye**
Not explained in the "install from source" page

### CODE REVIEW

**Comment by garply** [comment]
Hey admins! Can one of you let me know if it's safe to build this pull request? See https://alpha.sandstorm.io/grain/kB7bofATTL2jGuKSFwmqdA/GithubPRCommands for a list of commands.

**Comment by paulproteus** [comment]
Thanks @pdehaye this looks great. A few small suggestions:

- Add a `:` (or a `.`) at the end of the new "Get the sources..." line you added.

- I suggest saying "source code" rather than "sources", since in American English "source code" is more common, and the docs use the phrase 'source code' elsewhere.

Other than that, I think this is perfect. I can make those changes, or you can!

**Comment by paulproteus** [comment]
I'm very excited to merge this by the end of the day today, so if you want to fix these, great!

And if not, I can merge this about two hours from now and make the changes on top and update the docs.

File: docs/install.md [hunk 1]
```diff
@@ -107,13 +107,20 @@ Please install the following:
 On Debian or Ubuntu, you should be able to get all these with:
 
     sudo apt-get install build-essential libcap-dev xz-utils zip \
-        unzip imagemagick strace curl clang-3.4 discount
+        unzip imagemagick strace curl clang-3.4 discount git
     curl https://install.meteor.com/ | sh
 
+### Get the source code:
+
+Get the source code from the git repository
```
Comment (paulproteus) [inline docs/install.md]: This sentence needs a `:` or a `.` so that it's punctuated properly.

**Comment by paulproteus** [comment]
Also, good job being consistent with the Markdown style we're using (indenting things to make them code blocks).

I was going to suggest you prefix your commands with a `$` but it turns out this file doesn't use that! So you were more consistent than I was expecting!

File: docs/install.md (hunk 1 above)
Comment (paulproteus) [inline docs/install.md]: Oh I see! You added it to the section heading but I meant to the sentence below the section heading. So one more change requested:

remove the `:` from section heading, and add it to the sentence below the heading.

**Comment by paulproteus** [comment]
This is glorious. Thanks so much!

**Comment by paulproteus** [comment]
https://docs.sandstorm.io/en/latest/install/ is now up to date.

One confetti ball :confetti_ball: for you, @pdehaye , for submitting this and revising it so readily! Now it is _vastly_ easier to read the install page and then install Sandstorm from git.

**Pull request merged**